docs/quadratic/quadratic funding.md

```diff
@@ -0,0 +1,69 @@
+
+2. Democratization
+
+By enabling anyone to contribute, regardless of the size of their donation, Quadratic Funding democratizes the funding process. It empowers individuals to support projects they believe in, fostering a sense of ownership and engagement within the community.
+
+3. Transparency
+
+Transparency is fundamental to Quadratic Funding. All contributions and allocations are recorded on a public ledger, ensuring accountability and trust in the allocation process. This transparency also enables stakeholders to monitor the distribution of funds and hold administrators accountable for their decisions.
+
+
+# Quadratic Funding Round 
+
+A funding pool (budget) is established to match contributions made to eligible projects. This budget is typically funded by a combination of public and private sources, ensuring sustainability and continued support for public goods. Public goods are voted during the funding round governed by the quadratic-funding contract to ensure funding decisions are made collectively and transparently. Then, votes on proposals are matched using a quadratic formula, where the matching funds increase non-linearly with the number of unique voters. This means that smaller votes receive a higher matching ratio compared to larger ones, amplifying the impact of grassroots support.
+
+
+# Contracts
+
+There are two contracts, the quadratic funding contract and the NFT contract extending [`cw721-base`](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-base) for the use of non-transferable NFT owners instead of whitelist addresses for the following reasons:
+
+1. NFT ownership is recorded onchain which is transparent and immutable. On the contrary, whitelists may be stored off-chain, so NFT usage allows a more transparent and verifiable representation of ownership directly on the blockchain, ensuring greater visibility, accountability, and trust in the management.
+
+1. In addition, NFTs used during the funding round is interoperable in contrast to the whitelists so that they could be used for different projects or platforms that support the same NFT standard.
+
+1. NFT contract uses [`cw_ownable`](https://github.com/larry0x/cw-plus-plus/tree/main/packages/ownable) to assure only admin of the NFT contract can execute logic of the contract. NFTs are non-transferable so, the voting power associated with each NFT remains constant and is determined at the time of issuance (mint). This simplifies the voting power calculating as it does not change with time. Moreover, NFT ownership remains stable and is limited to the original recipient. This means that there is no risk of external parties acquiring NFTs solely for the purpose of influencing voting outcomes. This may enhance the integrity of the voting system.
+
+1. The voting power is stored in the metadata field for the use purpose in the quadratic funding formula. The metadata is in the following format:
+
+```rust
```

Comment:
This documentation is designed for a broader audience, not just developers, so it's advisable to avoid using technical jargon like "Rust" or terms like "Option" that might be confusing.